frontend/src/common/helpers/localStorageHelper.ts

```diff
@@ -0,0 +1,5 @@
+const setLocalStorageItem = (key: string, value: string): void => localStorage.setItem(key, value);
+
+const getLocalStorageItem = (key:string): string | null => localStorage.getItem(key);
+
+export { setLocalStorageItem, getLocalStorageItem };
```

Comment:
1. Call the file as storageHelper
2. The file should contain methods:
setAccessToken: (token: string) => void
getAccessToken: () => string